docs/content/getting-started/_index.md

```diff
@@ -92,17 +125,155 @@ class App extends Component {
+
+
+```
+<pfe-cta style="--pfe-cta--BackgroundColor:pink">
+```
+
+
+## 6. Victory!
+
+Be sure to browser test within your own site or application. If you run into any trouble, please file an issue in the [issue queue](https://github.com/patternfly/patternfly-elements/issues).
+
+<hr/>
+
+
+## FAQ
+
+### Should I use on=dark or color=darkest on my container? What's the difference?
+
+*   `on=dark` is being deprecated. Instead, custom classes already on the page should set broadcast values.
+*   The original goal for `on=dark` was all about context. But instead of having to provide another attribute somewhere, the card or a band or another container will inform any nested components that the text color needs to change through the set of broadcast variables. Existing broadcast vars (defined in `$BROADCAST-VARS`)  include: `text`, `ui-link`, `ui-link--hover`, `ui-link--visited`, `ui-link--focus`.
+*   If your page has a custom background color for a band or card, you should set the background and also set the broadcast colors.
+
+
+```
+.custom-dark-band {
+  background: black;
+  --pfe-broadcasted--color--text:             var(--pfe-theme--color--text--on-dark, #fff);
+  --pfe-broadcasted--color--ui-link:          var(--pfe-theme--color--ui-link--on-dark, #73bcf7);
+  --pfe-broadcasted--color--ui-link--hover:   var(--pfe-theme--color--ui-link--on-dark--hover, pink);
+  --pfe-broadcasted--color--ui-link--visited: var(--pfe-theme--color--ui-link--on-dark--visited, pink);
+  --pfe-broadcasted--color--ui-link--focus:   var(--pfe-theme--color--ui-link--on-dark--focus, pink);
+}
+```
+
+In themes (like the advanced-theme.css file from the Red Hat Theme ) we apply broadcast variables to plain links, because they are light DOM and also have default colors applied by the browser. This CSS file not only includes variables but also styles for headlines and links on the page. It sets the colors for these elements using a CSS variable, which web components can change the value of.
+
+For example, advanced-theme.css includes
+
+```
+.PFElement a {
+color: var(--pfe-broadcasted--color--ui-link);
+}
+```
+
+We choose not to apply broadcast colors to text elements like paragraphs because it still would not be high enough specificity to override anything coming from pre-existing stylesheets, and paragraphs will inherit color from parents. 
+
+
+```
+// this would not really be helpful to add to cp-theme or redhat-theme
+h1, h2, h3, h4, h5, h6, p { 
+  color: var(--pfe-broadcasted--color--text);
+}
+// if there was some class like this in the theme, It would override it anyway. 
+body.editorial .body1.generic1 {
+    color: #646464;
+}
+```
+
+Instead, in the host of components, use:
+
+```
+:host {
+  color: var(--pfe-broadcasted--color--text);
+}
+```
+
+Then call theme mixin to flip colors of the on=dark on=light attributes. Default tags will use these colors. If devs implementing the component have more specific styles on their page, they will have to handle it.
```

Comment:
Seeing how it is stated earlier that `on=dark` is being deprecated should this be updated to reflect that or do you want to wait till `on=dark` is officially not supported?

Reply:
removed, thanks!